aries-site/src/examples/templates/dashboards/content/chartCards/utils/window.js

```diff
@@ -7,4 +7,5 @@ export const defaultWindow = { begin: beginDate, end: endDate };
 export const REPORT_WINDOW_MAP = {
   'Last 30 Days': 30,
   'Last Year': 365,
+  Lifetime: 365 * 20,
```

Comment:
Is this just for sake of our examples to do 20 years? -- would we want to comment that this should likely be calculated based on the length of time an account had been active?